javascript-carousel/main.js

```diff
@@ -0,0 +1,53 @@
+var imageArray = ['images/001.png', 'images/004.png', 'images/007.png', 'images/025.png', 'images/039.png'];
+var $img = document.querySelector('img');
+var index = 0;
+function handleClick(event) {
+  var $imgUrl = $img.getAttribute('src');
+  if (event.target.className === 'fas fa-chevron-right fa-4x') {
+    clearInterval(intervalId);
+    intervalId = setInterval(myCallback, 3000);
+    changeImage($imgUrl, 'right');
+
+  } else if (event.target.className === 'fas fa-chevron-left fa-4x') {
+    clearInterval(intervalId);
+    intervalId = setInterval(myCallback, 3000);
+    changeImage($imgUrl, 'left');
+
+  } else if (event.target.className === 'far fa-circle') {
+    clearInterval(intervalId);
+    intervalId = setInterval(myCallback, 3000);
+    document.querySelector('.current').className = 'far fa-circle';
+    document.getElementById(event.target.id).className = 'current fas fa-circle';
+    $img.setAttribute('src', imageArray[event.target.id]);
+  }
+}
+
+function changeImage(imgUrl, direction) {
```

Comment:
You don't seem to be using the `imgUrl` parameter anymore, you should remove it and update all your function calls for it in the rest of your code 🤓